clique/src/protocols/mod.rs

```diff
@@ -59,6 +61,7 @@ impl<PK: PublicKey> Display for ProtocolError<PK> {
             NoParentConnection => write!(f, "cannot send result to service"),
             NoUserConnection => write!(f, "cannot send data to user"),
             NotAuthorized => write!(f, "peer not authorized"),
+            Timeout => write!(f, "network operation timed out"),
```

Comment:
Since this only happens during sending perhaps rename to `SendTimeout` and change the message appropriately? Alternatively (although possibly worse?) reuse `CardiacArrest` since the time you are waiting is very related to that anyway.

Reply:
Changed to `SendTimeout`.